orunmila.go

```diff
@@ -437,32 +433,34 @@ func addSubcmd(args []string) {
 	}
 	err = tx.Commit()
 	check(err)
+
+	db.Close()
```

Comment:
maybe move this to the the subcommand that opened the database. Here it will cause us problems down the road

Reply:
i always let the parent/caller to handle errors and free resources, today was an exception because what the hell was i thinking about XD

Reply:
btw the function which opened the db is `addSubcmd()` itself, no change will be done